.github/workflows/test_and_deploy.yml

```diff
@@ -59,7 +58,13 @@ jobs:
               cibuildwheel --print-build-identifiers --platform linux \
               | jq -nRc '{"dist": inputs, "os": "ubuntu-latest"}' \
               && cibuildwheel --print-build-identifiers --platform macos \
-              | jq -nRc '{"dist": inputs, "os": "macos-latest"}' \
+          | jq -Rc '
+            if (contains("x86_64")) then
+              {"dist": ., "os": "macos-13"}
+            else
+              {"dist": ., "os": "macos-latest"}
+            end
+          ' \
```

Comment:
Nice idea! This is about fixing the performance issues in arm builds right?

Reply:
Oh, I didn't check the performance on ARM.
It seems macos-latest + x86_64 is not supported and it always defaulted to macos-latest + ARM for that setting.
According to the compatibility table, we have to use macos-13 + x86_64 or macos-latest-large + x86_64:
https://github.com/actions/runner-images?tab=readme-ov-file#available-images

If there are no downsides to macos-latest-large, we could switch to the latter.

Reply:
Large runners are seemingly only available for enterprise customers.
Let's keep macos-13, then.